plugins/BEdita/Core/src/Model/Table/ObjectTypesTable.php

```diff
@@ -220,6 +230,21 @@ public function afterSave()
     }
 
     /**
+     * Don't allow delete actions if at least an object of this type exists.
+     *
+     * @param \Cake\Event\Event $event The beforeSave event that was fired
+     * @param \Cake\Datasource\EntityInterface $entity the entity that is going to be saved
+     * @return void
+     * @throws \Cake\Network\Exception\ForbiddenException if entity is not deletable
+     */
+    public function beforeDelete(Event $event, EntityInterface $entity)
+    {
+        if (TableRegistry::get('Objects')->find()->where(['object_type_id' => $entity->id])->count() > 0) {
```

Comment:
`count()` triggers a count of how many rows of that kind exist. It should be fast, but we're not interested in the actual amount of rows exist — it's enough to check if one exist.

We might use `TableRegistry::get('Objects')->exists([/* ... */])` instead. It should be faster since it leverages the load on database, and possibly be even more self-explanatory.